### ISSUE TITLE
Prism formating issue on JS code that create formatter

### ISSUE THREAD

**Issue by jcubic** [jcubic/jquery.terminal#523]
### Expected behavior

JS code should be highlighted properly

### Actual behavior

JS code is brokan

### Steps to reproduce

Try to format this javascript with prism:

```javascript
    $.terminal.defaults.formatters.push(function(string) {
        return string.replace(/([^`]|^)`([^`]+)`([^`]|$)/g, '$1[[b;#fff;]$2]$3');
    });
```

it's part of `chat.js` on terminal website. (formatting of text inside back ticks).

You can see this with less command with this demo https://codepen.io/jcubic/pen/zEyxjJ

and type:

`less https://terminal.jcubic.pl/js/chat.js `

**Comment by jcubic** [comment]
Prism have no problems in rendering of regex with backticks so this is problem in less.js and escaping.
```
term.echo($.terminal.prism('javascript', '/xxx`xx`yyy/'))
```
this works.

**Comment by jcubic** [comment]
This is also broken, (this should be tested):
```javascript
$.get('https://terminal.jcubic.pl/js/chat.js').then((text) => {
   term.echo($.terminal.prism('javascript', $.terminal.escape_brackets(text)));
});
```

Not sure if you can make formatters in source code work with prism (such meta).

**Comment by jcubic** [comment]
fixed in #410 by 17d903f7b68e3255f361deca99f9ff02fc281455

There is one not related issue with wrapping though.
